### ISSUE TITLE
Versioning requirements in lar packages

### ISSUE THREAD

**Issue by nathanielerowe** [LArSoft/larsoft-spack-recipes#7]
I was recently working with a spack build of sbndcode@09_91_02_01 and found that some of their .fcl files could not run due to the addition of PFPModuleLabel in larreco's GnocchiCalorimetry. This was due to sbncode and larsoft being out of step with larreco in our spack build. 

I was able to rebuild using the appropriate larreco version, and the .fcl file now runs, but is there a plan to put in stricter versioning requirements in the default larsoft spack recipes to avoid similar scenarios in the future? Or is the intent for the user to specify specific versions of all "lar" packages when concretizing? Was mainly just wondering if there was an agreed upon solution to this general problem that had not yet been implemented.

**Comment by gartung** [comment]
The versions are specified in the environment yaml used for the environment build.

**Comment by nathanielerowe** [comment]
Yeah this was from a while back, but thanks for following up. My original concerns about this from the experiment perspective were addressed in the [document released in December](https://indico.fnal.gov/event/67311/contributions/304887/attachments/183733/252612/plan-for-standard-builds.pdf).
